### DISCUSSION TITLE
Multiple metadata values with the same key on the same task

### DISCUSSION THREAD

**Discussion by evanp** [todotxt/todo.txt#55] [Ideas]
I think that this is OK, according to the current spec:

```
2018-03-15 Multiple metadata values with the same key on the same task label:question label:invalid
```

Nothing in the spec currently says that the key names need to be unique in a single task. Is that everyone else's understanding, too? I know that widely-used keys like `due` and `t` are typically unique on a line, but I could see using metadata for other data like `tag` or `with-person`.

**Comment by TangentFoxy** [comment]
I've made the same assumption. It makes sense for some things, while not for all.

As an example where it can make sense, say I put two copy-pasted links into a single note. I end up with two `http` or `https` tags.
